protocol-specs/generator_outputs/unlock_transaction/unlock_transaction.json

```diff
@@ -0,0 +1,235 @@
+{
+	"title": "Valid unlock transaction",
+	"summary": "Valid unlock transaction",
+	"config": {
+		"network": "devnet"
+	},
+	"runner": "unlock_transaction",
+	"handler": "unlock_transaction",
+	"testCases": {
+		"input": {
+			"account": {
+				"passphrase": "lava toe nuclear candy erode present guilt develop include type pluck current",
+				"publicKey": "8c3d81b1555fbe4692adfa1026ee21c043633b9369924cf2790e2e0fc6b47a66",
+				"address": "841832338348093031L"
+			},
+			"networkIdentifier": "e48feb88db5b5cf5ad71d93cdcd1d879b6d5ed187a36b0002cc34e0ef9883255",
+			"delegates": [
```

Comment:
For a valid unlock transaction, there must be a valid vote transaction with negative vote weight. But the state here shows `account`, `networkIdentifier` and `delegates` as input, we don't have the information of negative votes then how this unlock transaction can be valid?

Reply:
For unlock transaction to be valid, it needs unlocking objects in the account, but the state information will be created in the unit test